hack/jenkins/release_build_and_upload.sh

```diff
@@ -136,5 +127,19 @@ do
   docker rmi -f ${IMAGE_NAME}
 done
 
+
+# upload to google bucket
+gsutil -m cp out/* "gs://$BUCKET/releases/$TAGNAME/"
```

Comment:
there might be things in out folder that we dont want ? maybe like Linters binaries or somethign like that ? and why this is added to this PR? before this PR we didnt have artifacts in the GCS bucket?

Reply:
nevermind we are just moving this code